modules/ocf/manifests/packages/chrome.pp

```diff
@@ -1,4 +1,8 @@
 class ocf::packages::chrome {
+  class { 'ocf::packages::chrome::apt':
```

Comment:
the apt changes needs to happen in `stage => first` and puppet kinda sucks

(we used to follow this same pattern for percona until last night)